octoprint_mrbeam/util/github_api.py

```diff
@@ -0,0 +1,56 @@
+# TODO move to util an refactor cloud config part to use same
+import base64
```

Comment:
Please add a proper doc string at the the top of the file

Reply:
What is a proper doc string for such files? We didn't use them so far

Reply:
I believe when I started reviewing there was no doc string here at the top. Now this one is OK